### ISSUE TITLE
Ring Bell does not at Gates of Hades does nothing

### ISSUE THREAD

**Issue by wesleyneal346** [devshane/zork#9]
I try to ring the bell outside the Gates of Hades and nothing happens. I was under the impression that ringing the bell would make it glow red hot and the spirits would get scared of me. Ive tried lighting the candles and reading the good book and nothing happens there either. Is this a bug or am I missing something?

```
You are outside a large gateway, on which is inscribed:

        "Abandon every hope, all ye who enter here."

The gate is open.  Through it you can see a desolation, with a pile of
mangled corpses in one corner.  Thousands of voices, lamenting some
hideous fate, can be heard.
The way through the gate is barred by evil spirits, who jeer at your
attempts to pass.
>ring bell
Ding, dong.
>light match
One of the matches starts to burn.
>light candles with match
The candles are lit.
The match has gone out.
>turn off lamp
The lamp is now off.
>read book
                Commandment #12592
Oh ye who go about saying unto each other:  "Hello sailor":
Dost thou know the magnitude of thy sin before the gods?
Yea, verily, thou shalt be ground between two stones.
Shall the angry gods cast thy body into the whirlpool?
Surely, thy eye shall be put out with a sharp stick!
Even unto the ends of the earth shalt thou wander and
unto the land of the dead shalt thou be sent at last.
Surely thou shalt repent of thy cunning.
>e
Some invisible force prevents you from passing through the gate.
>odysseus
Wasn't he a sailor?
>ring bell
Ding, dong.
>open bible
The book is open to page 569.
>close bible
As hard as you try, the book cannot be closed.
>version
V2.7A
```

**Comment by wesleyneal346** [comment]
I found the original map but found it hard to read. So I basically GIMP'ed a new map after playing most of the game and found all the treasure I could while following the original as close as i could. I also have a "Scripted Moves" file that helps for people who have played the game a bit. It's kind of like a short cut. I've tested the scripts against this repo (V2.7A) in Cygwin and Windows command Prompt.

Overall, I've found the game to be passable, however, some of the game's functionality is missing. Apparently, there is a version V3.2B, but I have yet to find the source code for it.

[Scripted Moves.txt](https://github.com/devshane/zork/files/1519856/Scripted.Moves.txt)
![dungeon-1981](https://user-images.githubusercontent.com/22687849/33460749-d528f902-d5ec-11e7-8fe2-ecaf1d00e56f.png)
